addons/website/static/tests/tour_utils/widget_lifecycle_dep_widget.js

```diff
@@ -0,0 +1,28 @@
+/** @odoo-module **/
+
+import publicWidget from "web.public.widget";
+
+window.localStorage.WidgetAndWysiwygLifecycle = window.localStorage.WidgetAndWysiwygLifecycle || "[]";
+
+publicWidget.registry["countdown_patch"] = publicWidget.Widget.extend({
+    selector: ".s_countdown",
+    disabledInEditableMode: false,
+    /**
+     * @override
+     */
+    async start() {
+        window.localStorage.WidgetAndWysiwygLifecycle = JSON.stringify(
+            JSON.parse(window.localStorage.WidgetAndWysiwygLifecycle).concat("widgetStart"));
+        await this._super(...arguments);
+        this.$target[0].classList.add("public_widget_started");
+    },
+    /**
+     * @override
+     */
+    destroy() {
+        this._super(...arguments);
+        this.$target[0].classList.remove("public_widget_started");
+        window.localStorage.WidgetAndWysiwygLifecycle = JSON.stringify(
+            JSON.parse(window.localStorage.WidgetAndWysiwygLifecycle).concat("widgetStopped"));
+    },
```

Comment:
Consistency: you named the event "widgetStart" and "widgetStopped". Choose: either ("widgetStarted" and "widgetStopped") or ("widgetStart" and "widgetStop")